Analytics/ElasticTelemetry.cs

```diff
@@ -25,6 +25,13 @@ private ElasticTelemetry()
                 //Load custom app.config for this assembly
                 var appConfig = ConfigurationManager.OpenExeConfiguration(Assembly.GetExecutingAssembly().Location); 
 
+                if (appConfig.AppSettings.Settings["SmtpServer"].Value == "your.smtp.server")
```

Comment:
It will works if the configuration is still the default.
But I think we should also manage the case where the configuration file is wrong.
Like someone put  `http://BadServer.com` in `ElasticServer` property.

Reply:
As `SmtpServer` is not used by ElasticAnalytics, I'd prefer that this test use an `ElasticXXX` property.